Tests/Clappr_Tests/AVFoundationPlaybackTests.swift

```diff
@@ -240,9 +257,9 @@ class AVFoundationPlaybackTests: QuickSpec {
                     var didCallChangeDvrAvailability: Bool?
                     let playerAsset = AVURLAssetStub(url: URL(string: "http://clappr.sample/master.m3u8")!)
                     
-                    func setupTest(minDvrSize: Double, seekableTimeRange: Double) {
+                    func setupTest(minDvrSize: Double, seekableTimeRange: Double, duration: CMTime = CMTime.indefinite) {
```

Comment:
I think you can remove the prefix of `CMTime.indefinite`, leaving only the `.indefinite`.

Reply:
func setupTest(minDvrSize: Double, seekableTimeRange: Double, duration: CMTime = .indefinite) {

Reply:
Done.